src/amo/components/Home/index.js

```diff
@@ -129,12 +141,27 @@ export class HomeBase extends React.Component {
   }
 
   render() {
-    const { i18n } = this.props;
+    const { i18n, popularExtensions } = this.props;
 
     return (
       <div className="Home">
```

Comment:
This needs `{errorHandler.renderErrorIfPresent()}`

Reply:
Shoot, I noticed that too and must've forgot to hit submit on the comment 😭

Reply:
ooops, will add it in the next PR, sorry for that I forgot it.